compiler/damlc/tests/src/unstable-types.sh

```diff
@@ -47,6 +47,7 @@ for LF_VERSION in $PKG_DB/*; do
 "DA.Generics:K1"
 "DA.Generics:Par1"
 "DA.Generics:U1"
+"DA.Internal.Desugar:Consuming"
```

Comment:
This isn’t right, we should update `consumingTypes` in `DA.Daml.LFConversion` to filter this out.

Reply:
I see. Will fix.